apis/ec2/manualv1alpha1/common.go

```diff
@@ -538,6 +546,18 @@ type InstanceNetworkInterfaceSpecification struct {
 	// +optional
 	IPv6Addresses []InstanceIPv6Address `json:"ipv6Addresses"`
 
+	// The number of IPv6 delegated prefixes to be automatically assigned to the
+	// network interface. You cannot use this option if you use the Ipv6Prefix option.
+	// +optional
+	// +immutable
+	Ipv6PrefixCount *int32 `json:"ipv6PrefixCount"`
+
+	// One or more IPv6 delegated prefixes to be assigned to the network interface. You
+	// cannot use this option if you use the Ipv6PrefixCount option.
+	// +optional
+	// +immutable
+	Ipv6Prefixes []Ipv6PrefixSpecificationRequest `json:"ipv6Prefixes"`
```

Comment:
```suggestion
	Ipv6Prefixes []Ipv6PrefixSpecificationRequest `json:"ipv6Prefixes,omitempty"`
```

Optional types have omitempty tag.